schedule/yast/sle/flows/default_sle15sp3_s390x_kvm.yaml

```diff
@@ -0,0 +1,48 @@
+---
+# Default ordered sequence of steps for sle15sp3 with maintenance updates
+bootloader:
+  - installation/bootloader_start
+setup_libyui:
+  - installation/setup_libyui
+product_selection:
+  - installation/product_selection/install_SLES
+license_agreement:
+  - installation/licensing/accept_license
+registration:
+  - installation/registration/register_via_scc
+extension_module_selection:
+  - installation/module_registration/register_extensions_and_modules
+add_on_product:
+  - installation/add_on_product/add_maintenance_repos
+  - installation/add_on_product_installation/accept_add_on_installation
+system_role:
+  - installation/system_role/accept_selected_role_SLES_with_GNOME
+suggested_partitioning:
+  - installation/partitioning/accept_proposed_layout
+clock_and_timezone:
+  - installation/clock_and_timezone/accept_timezone_configuration
+local_user:
+  - installation/authentication/use_same_password_for_root
+  - installation/authentication/default_user_simple_pwd
+software:
+  - installation/select_patterns
+booting:
+  - installation/bootloader_settings/disable_boot_menu_timeout
+installation_settings:
+  - installation/launch_installation
+installation:
+  - installation/confirm_installation
+  - installation/performing_installation/perform_installation
+stop_timeout_system_reboot:
+  - installation/performing_installation/stop_timeout_system_reboot_now
+installation_logs:
+  - installation/logs_from_installation_system
+confirm_reboot:
+  - installation/performing_installation/confirm_reboot
+  - installation/performing_installation/reconnect_after_reboot
+handle_reboot:
+  - installation/handle_reboot
```

Comment:
looks like this one we could group it above and have:
```
reboot:
  - installation/performing_installation/confirm_reboot
  - installation/performing_installation/reconnect_after_reboot
  - installation/handle_reboot
```

Reply:
Done